<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Seleccionar el directorio donde se encuentran los espectros</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QToolButton" name="tlb_open">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLineEdit" name="le_path"/>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Guardar archivo ordenado en...:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QToolButton" name="tlb_save">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="QLineEdit" name="le_save_path"/>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Ordenar espectros utilizando su:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QRadioButton" name="rb_exp_time">
      <property name="text">
       <string>Tiempo de Exposicion</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QRadioButton" name="rb_counter">
      <property name="text">
       <string>Posicion del Contador</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QRadioButton" name="rb_red">
      <property name="text">
       <string>Red</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="btn_sort">
      <property name="text">
       <string>Sort</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
